spec/Appendix B -- Grammar Summary.md

```diff
@@ -412,3 +419,21 @@ TypeSystemDirectiveLocation : one of
 - `ENUM_VALUE`
 - `INPUT_OBJECT`
 - `INPUT_FIELD_DEFINITION`
+
+SchemaCoordinate :
+
+- TypeCoordinate
+- MemberCoordinate
+- ArgumentCoordinate
+- DirectiveCoordinate
+- DirectiveArgumentCoordinate
+
+TypeCoordinate : Name
+
+MemberCoordinate : Name . Name
+
+ArgumentCoordinate : Name . Name ( Name : )
```

Comment:
Any reason why `type.field(arg:)` was preferred over `type.field(arg)`? 

Apologies if this has already been discussed before.

Reply:
We went back and forth on this a few times when this was first proposed. Personally I don't care either way (slightly prefer `type.field(arg)` even) but `arg:` mirrors other languages (e.g.)
```suggestion
ArgumentCoordinate : Name . Name ( Name : )https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions/#:~:text=The%20body%20of%20the%20greet(person%3A)
```